utils/rpms/daos.spec

```
@@ -227,14 +235,13 @@ scons %{?_smp_mflags}      \
       --install-sandbox=%{buildroot} \
       %{buildroot}%{_prefix}         \
       %{buildroot}%{conf_dir}        \
-      USE_INSTALLED=all               \
       CONF_DIR=%{conf_dir}            \
       PREFIX=%{_prefix}               \
       %{?scons_args}                  \
       %{?compiler_args}
 
 %if ("%{?compiler_args}" == "COMPILER=covc")
-cp test.cov %{buildroot}/usr/lib/daos/TESTING/ftest/
+cp -f /tmp/test.cov %{?buildroot}/usr/lib/daos/TESTING/ftest/
```

`%{buildroot} is not optional:
```suggestion
cp -f /tmp/test.cov %{buildroot}/usr/lib/daos/TESTING/ftest/
```

---

ah thanks